azure-functions/acbs-function/api-dtfs-cental-api.js

```diff
@@ -0,0 +1,51 @@
+const axios = require('axios');
+
+const centralAPI = process.env.DTFS_CENTRAL_API;
+
+require('dotenv').config();
+
+const getDeal = async (dealId) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${centralAPI}/v1/tfm/deals/${dealId}`,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return response.data.deal;
+  } catch (error) {
+    console.error(`ACBS - Error while fetching deal ${dealId}.`, { error });
+    return false;
+  }
+};
+
+const createActivity = async (dealId, activity) => {
```

Comment:
Should we call this `updateTfmDeal` and then pass the`tfm.activities`object in from wherever it's called?

Just then if we need to update a TFM deal from somewhere else in ACBS, can just call this function. ?

Also should we call `getDeal` `getTfmDeal`so it's clearer?